x/cheqd/tests/deactivate_did_test.go

```diff
@@ -0,0 +1,81 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/cheqd/cheqd-node/x/cheqd/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeactivateDid(t *testing.T) {
+	keys := map[string]KeyPair{
+		AliceKey1:    GenerateKeyPair(),
+		AliceKey2:    GenerateKeyPair(),
+		BobKey1:      GenerateKeyPair(),
+		BobKey2:      GenerateKeyPair(),
+		BobKey3:      GenerateKeyPair(),
+		BobKey4:      GenerateKeyPair(),
+		CharlieKey1:  GenerateKeyPair(),
+		CharlieKey2:  GenerateKeyPair(),
+		CharlieKey3:  GenerateKeyPair(),
+		CharlieKey4:  GenerateKeyPair(),
+		ImposterKey1: GenerateKeyPair(),
+	}
+
+	cases := []struct {
+		valid      bool
+		name       string
+		signerKeys []SignerKey
+		signers    []string
+		msg        *types.MsgDeactivateDidPayload
+		errMsg     string
+	}{
+		{
+			valid: true,
+			name:  "Valid: Key rotation works",
```

Comment:
Key rotation is impossible for Deactivation tx. Changing the test name is needed